Formula/h/helm.rb

```diff
@@ -1,27 +1,33 @@
 class Helm < Formula
   desc "Kubernetes package manager"
   homepage "https://helm.sh/"
-  url "https://github.com/helm/helm.git",
-      tag:      "v3.15.2",
-      revision: "1a500d5625419a524fdae4b33de351cc4f58ec35"
+  url "https://github.com/helm/helm/archive/refs/tags/v3.15.3.tar.gz"
+  sha256 "f4eb30fa8285091ceffba16cf5ad7ec4b444897bd1df615af4bef3842f6d6d0e"
   license "Apache-2.0"
   head "https://github.com/helm/helm.git", branch: "main"
 
   bottle do
-    sha256 cellar: :any_skip_relocation, arm64_sonoma:   "acf31d718a351224ceddc070b77c88bfda41fdeca0568bc1fb73933fd76247c2"
-    sha256 cellar: :any_skip_relocation, arm64_ventura:  "71847de7550d6ce1179d0fe8ec93f737416db04b88db113bfcb49e0226aadff5"
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "2e15df753d03f6aa213bcc3c204b49da112f8ba16a38f2e563261f7b7df41c01"
-    sha256 cellar: :any_skip_relocation, sonoma:         "aad9245f76cf88b3872c04b2e33d63c4a7c860974bc97a0928901ea084da0a1c"
-    sha256 cellar: :any_skip_relocation, ventura:        "83af0d48b3daf8b926b9ff6f8a911513233b8afc2766b434bf66181849a9737d"
-    sha256 cellar: :any_skip_relocation, monterey:       "efe450f4ec2da4df86d1832422e13239f3ef1ca87a875b291813668f91864573"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "2c515f4142a5e45df24a30eecbda7a0bf36a285f040983a4c2ee0132e3900732"
+    sha256 cellar: :any_skip_relocation, arm64_sonoma:   "827f6a04199af59844dfecb346ab096a2684b1c7dc0a4b82f6c155842524dc7d"
+    sha256 cellar: :any_skip_relocation, arm64_ventura:  "75e248744bdf1bd99f85bca02a12f6f57550d60a59bd814ece38a330ee52d741"
+    sha256 cellar: :any_skip_relocation, arm64_monterey: "84e42ca0cc01fa6e9dca03cfcfede9b02318d13fe81cb1b2be42e0bc4b25d0cc"
+    sha256 cellar: :any_skip_relocation, sonoma:         "e14a0b990ea7ceac59c6e33c8565ebcd89430b9d54fb697443419ea6db4f2e0a"
+    sha256 cellar: :any_skip_relocation, ventura:        "df7fc5b133e2038d4f7805775a59442032f3ff3b7dea27aa36ea6b415df194a0"
+    sha256 cellar: :any_skip_relocation, monterey:       "d6a4d28fa4ab4c2d832bf0a5911de6091172aa0546bae32338bfcf7ece068457"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:   "dbdc826f25219deb0187c808ff576a84ea69527d887f5cd4848f8bfe7ee4a411"
   end
 
   depends_on "go" => :build
 
   def install
-    system "make", "build"
-    bin.install "bin/helm"
+    ldflags = %W[
```

Comment:
The makefile also disable cgo and sets some annotations for the k8s client version: https://github.com/helm/helm/blob/2feac15cc3252c97c997be2ced1ab8afe314b429/Makefile#L61-L69

Reply:
@chenrui333 we need to revert this pls: https://github.com/helm/helm/issues/13174

Reply:
i see, make sense 👍